README.md

```diff
@@ -142,3 +145,16 @@ The following users have deployed JanusGraph in production.
 * [Red Hat](https://www.redhat.com/) - [application migration and assessment tool](https://developers.redhat.com/products/rhamt/overview/) built on [Windup](https://github.com/windup/windup)
 * [Times Internet](https://timesinternet.in)
 * [Uber](https://uber.com)
+
+## Cite
+
```

Comment:
(nitpick) Maybe we should add a short sentence here to explain this section, especially for people who are not familiar with academic work?
Something like this maybe:
_If you use JanusGraph in your research, please cite it using this metadata:_

This is mostly the formulation TensorFlow uses: https://github.com/tensorflow/tensorflow/blob/master/CITATION.cff